src/agent/agent.js

```diff
@@ -22,8 +22,8 @@ export class Agent {
 
         initModes(this);
 
-        this.bot.on('login', async () => {
-            console.log(`${this.name} logged in.`);
+        this.bot.once('spawn', async () => {
```

Comment:
What effects does this have?

Reply:
on login, the bot had not spawned yet so stats/inventory were null. this is triggered on spawn, only once.